geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/DiskStoreCommandsJUnitTest.java

```diff
@@ -103,339 +100,111 @@ private DiskStoreDetails createDiskStoreDetails(final String memberId,
-
-    final ResultCollector mockResultCollector =
-        mockContext.mock(ResultCollector.class, "ResultCollector");
-
-    mockContext.checking(new Expectations() {
-      {
-        oneOf(mockFunctionExecutor).setArguments(with(equal(diskStoreName)));
-        will(returnValue(mockFunctionExecutor));
-        oneOf(mockFunctionExecutor).execute(with(aNonNull(DescribeDiskStoreFunction.class)));
-        will(returnValue(mockResultCollector));
-        oneOf(mockResultCollector).getResult();
-        will(returnValue(Collections.singletonList(new Object())));
-      }
-    });
-
-    final DescribeDiskStoreCommand describeCommand =
-        createDescribeDiskStoreCommand(mockCache, mockMember, mockFunctionExecutor);
-
-    try {
-      describeCommand.getDiskStoreDescription(memberId, diskStoreName);
-    } catch (RuntimeException expected) {
-      assertEquals(
-          CliStrings.format(CliStrings.UNEXPECTED_RETURN_TYPE_EXECUTING_COMMAND_ERROR_MESSAGE,
-              Object.class.getName(), CliStrings.DESCRIBE_DISK_STORE),
-          expected.getMessage());
-      assertNull(expected.getCause());
-      throw expected;
-    }
+    final String diskStoreName = "mockDiskStore";
+    when(mockFunctionExecutor.execute(any(DescribeDiskStoreFunction.class)))
+        .thenReturn(mockResultCollector);
+    when(mockResultCollector.getResult()).thenReturn(Collections.singletonList(new Object()));
+
+    assertThatThrownBy(
+        () -> describeDiskStoreCommand.getDiskStoreDescription(memberId, diskStoreName))
+            .isInstanceOf(RuntimeException.class).hasNoCause().hasMessage(
+                CliStrings.format(CliStrings.UNEXPECTED_RETURN_TYPE_EXECUTING_COMMAND_ERROR_MESSAGE,
+                    Object.class.getName(), CliStrings.DESCRIBE_DISK_STORE));
   }
 
   @Test
   public void testGetDiskStoreList() {
-    final InternalCache mockCache = mockContext.mock(InternalCache.class, "InternalCache");
-
-    final DistributedMember mockDistributedMember =
-        mockContext.mock(DistributedMember.class, "DistributedMember");
-
-    final AbstractExecution mockFunctionExecutor =
-        mockContext.mock(AbstractExecution.class, "Function Executor");
-
-    final ResultCollector mockResultCollector =
-        mockContext.mock(ResultCollector.class, "ResultCollector");
-
-    final DiskStoreDetails diskStoreDetails1 =
-        createDiskStoreDetails("memberOne", "cacheServerDiskStore");
-    final DiskStoreDetails diskStoreDetails2 =
-        createDiskStoreDetails("memberOne", "gatewayDiskStore");
     final DiskStoreDetails diskStoreDetails3 = createDiskStoreDetails("memberTwo", "pdxDiskStore");
     final DiskStoreDetails diskStoreDetails4 =
         createDiskStoreDetails("memberTwo", "regionDiskStore");
-
+    final DiskStoreDetails diskStoreDetails2 =
+        createDiskStoreDetails("memberOne", "gatewayDiskStore");
+    final DiskStoreDetails diskStoreDetails1 =
+        createDiskStoreDetails("memberOne", "cacheServerDiskStore");
```

Comment:
Not a problem per se, but this is an odd reordering of the `DiskStoreDetails` declarations.